src/main/java/CalculatorInputString.java

```diff
@@ -0,0 +1,20 @@
+public class CalculatorInputString {
+	public static final String MESSAGE_VALUE_IS_NOT_NUMBER_FORMAT = "VALUE_IS_NOT_NUMBER_FORMAT";
+	private final String value;
```

Comment:
```suggestion
	public static final String MESSAGE_VALUE_IS_NOT_NUMBER_FORMAT = "VALUE_IS_NOT_NUMBER_FORMAT";
	
	private final String value;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)

Reply:
유념하겠습니다!